keps/sig-node/3960-pod-lifecycle-sleep-action/kep.yaml

```diff
@@ -0,0 +1,40 @@
+title: "Pod lifecycle sleep action"
+kep-number: 3960
+authors:
+  - "@AxeZhan"
+owning-sig: sig-node
+participating-sigs: []
+status: provisional
+creation-date: 2023-04-22
+reviewers:
+  - "@thockin"
+approvers:
+  - "@sig-node-leads"
+see-also: []
+replaces: []
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.29"
```

Comment:
so you are NOT aiming for this next release (1.28) but the one after it?

Reply:
Well, as Enhancements Freeze is coming soon, I'm a little afraid if we can catch up. If we reach an agreeement on this kep, sure I can implemenet this in 1.28. :)

Reply:
I'll approve it as-is, but you should follow up to change it to 28